xbmc/interfaces/legacy/ModuleXbmc.h

```diff
@@ -404,6 +404,14 @@ namespace XBMCAddon
     void audioResume();
 
     /**
+     * getUserAgent() -- Returns Kodis HTTP UserAgent string
```

Comment:
I've changed it, but think Kodi's is wrong.
I'm no native speaker, but doesn't "Kodi's" mean Kodi is/has the UA string
while "Kodis" denotes the possession of the UA string.